sigar_linux_common_test.go

```diff
@@ -0,0 +1,101 @@
+package gosigar_test
```

Comment:
I think we need the same build tags as in `sigar_linux_common.go`, so this is not executed when running in other operating systems.
```suggestion
package gosigar_test

//go:build freebsd || linux
// +build freebsd linux
```

Reply:
i added only linux (the test does not test the freebsd branch, i have no freebsd to verfiy mtab there)